chart/k8gb/templates/external-dns/external-dns-route53.yaml

```diff
@@ -36,6 +26,7 @@ spec:
         - --aws-zone-type=public # only look at public hosted zones (valid values are public, private or no value for both)
         - --registry=txt
         - --txt-owner-id= {{ .Values.route53.hostedZoneID }}
+        - --log-level=debug # debug only
```

Comment:
debug by default?

Reply:
Looks like leftover from debugging

Reply:
@k0da yes, by default. It's not super verbose still useful. We are using the same for coredns external-dns instance.

Reply:
@ytsarev worth to mention it in a commit message then

Reply:
@k0da sorry, missed this one, will document properly during next iteration